pkg/cli/mirror/list/list.go

```diff
@@ -0,0 +1,37 @@
+package list
+
+import (
+	"github.com/RedHatGov/bundle/pkg/cli"
+	"github.com/spf13/cobra"
+	kcmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+func NewListCommand(f kcmdutil.Factory, ro *cli.RootOptions) *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List available platform and operator content and their version",
+		Example: templates.Examples(`
+			# Output default operator channles
+			oc-mirror list operators --channels
+			# Output OCP release channel list
```

Comment:
This would be easier to read with spacings in between commands and comments, ex
```suggestion
			oc-mirror list operators --channels

			# Output OCP release channel list
```